src/index.js

```diff
@@ -6,7 +6,24 @@ const {quality,
                 signingClaimData,
                 multiSigningData,
                 binaryToJSON,
-                serializeObject}} = coreTypes;
+                serializeObject,
+                BinaryParser}} = coreTypes;
+
+function decodeLedgerData(binary) {
+  assert(typeof binary === 'string', 'binary must be a hex string');
```

Comment:
I suggested ledgerData because when you query rippled with the binary flag on, it returns the field `ledger_data` with the encoded header - but it doesn't matter much to me